.gitpod.yml

```
@@ -1,24 +1,24 @@
 image:
   file: .gitpod.dockerfile
 ports:
-- port: 3000 # Theia
-- port: 3030 # VS Code extension tests
-- port: 9229 # Node.js debug port
-  onOpen: ignore
-- port: 9339 # Node.js debug port
-  onOpen: ignore
-- port: 6080
-  onOpen: ignore
-- port: 5900
-  onOpen: ignore
+  - port: 3000 # Theia
+  - port: 3030 # VS Code extension tests
+  - port: 9229 # Node.js debug port
+    onOpen: ignore
+  - port: 9339 # Node.js debug port
+    onOpen: ignore
+  - port: 6080
+    onOpen: ignore
+  - port: 5900
+    onOpen: ignore
```

On a side-note, it would be cool to have these ports in numerical order:

```suggestion
  - port: 3000 # Theia
  - port: 3030 # VS Code extension tests
  - port: 5900
    onOpen: ignore
  - port: 6080
    onOpen: ignore
  - port: 9229 # Node.js debug port
    onOpen: ignore
  - port: 9339 # Node.js debug port
    onOpen: ignore
```